www/index.html

```diff
@@ -90,9 +90,12 @@ <h5 class="modal-title" id="modalLabel">Modal title goes here</h5>
                     <!-- Collect the nav links, forms, and other content for toggling -->
                     <div class="collapse navbar-collapse" id="collapsibleNavbar">
                         <ul class="navbar-nav">
-                            <li id="liHomeNav" class="nav-item"><button type="button" class="btn btn-light" id="btnHome">Home</button></li>
-                            <li id="liConfigureNav" class="nav-item"><button type="button" class="btn btn-light" id="btnConfigure">Configure</button></li>
-                            <li id="liAboutNav" class="nav-item"><button type="button" class="btn btn-light" id="btnAbout">About</button></li>
+                            <li id="liHomeNav" class="nav-item"><button type="button" class="btn btn-light" id="btnHome"><i class="fa fa-home"></i></button></li>
+                            <li id="liConfigureNav" class="nav-item"><button type="button" class="btn btn-light" id="btnConfigure"><i class="fa fa-wrench"></i></button></li>
+                            <li id="liAboutNav" class="nav-item"><button type="button" class="btn btn-light" id="btnAbout"><i class="fa fa-info"></i></button></li>
+                            <li><span class="btn-group">
```

Comment:
Why have you placed this button inside a `btn-group` span? Is there a reason for that? I can't see how this button is different from the others in the same row.